java/README.md

```diff
@@ -1,14 +1,42 @@
-Prerequisites
--------------
+# CuVS Java API
 
-* JDK 22
-* Maven 3.9.6 or later
+CuVS Java API attempts to provide a Java based simple, efficient, and a robust vector search API.
 
-To build this API, please do `./build.sh java` in the top level directory. Since this API is dependent on `libcuvs` it must be noted that `libcuvs` gets built automatically before building this API.
+> [!CAUTION]
+> This is an experimental release and updates to this API should be expected in the coming days.
```

Comment:
I've learned that timeboxing like this is not a good idea. We *should* be clear that APIs are experimental, but they'll be this way for the 25.02 release. Great if we can remove them in the 25.04 release, but this stable release will have experimental APIs. 

```suggestion
> cuVS 25.02 contains an experimental version and updates to this API are expected in the coming release.
```